### ISSUE TITLE
Piste:type should mention regional signs/colors

### ISSUE THREAD

**Issue by rugk** [openstreetmap/iD#4856]
Currently when editing/adding a piste the selection for the `piste:type` mentions things as "green circle", blue square" etc.

**Comment by bhousel** [comment]
> However that only applies for the US

I was expecting that people would just remove any US-specific text when they do the translations.
e.g. In the German translation, they could just remove those and replace them with whatever they do in Germany.


For reference:

<img width="377" alt="screenshot 2018-03-07 13 39 48" src="https://user-images.githubusercontent.com/38784/37111186-11fc630c-220d-11e8-8b8f-dd6d370911e6.png">

**Comment by rugk** [comment]
Yeah, and the German translation is just literally, here. Maybe you need some translation instructions/notes there. It seems it was not obvious for the translators to change that.

Also, BTW, what happens when US users do some armchair mapping or similar and change stuff in Europe?

This may be a more general question, as it may of course be an unlike case for this issue.

**Comment by bhousel** [comment]
> Also, BTW, what happens when US users do some armchair mapping or similar and change stuff in Europe?

I hope people aren't armchair mapping piste difficulty 😆

**Comment by rugk** [comment]
Sure, that's why I said, this may be a general question.

**Comment by manfredbrandl** [comment]
I just improved the German translation. Thanks for the hint, it was translated literally before
